frontend/src/lib/events.ts

```diff
@@ -0,0 +1 @@
+export const NEW_CONSENT_REQUEST = 'new_consent_request'
```

Comment:
nitpick (non-blocking): if we are going to have multiple events possibly in future I'd prefer this to be a map